src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderRepository.java

```diff
@@ -29,4 +29,7 @@ default Optional<Offender> findOffenderByNomsId(String nomsId){
     @Query(value =
         "select o from Offender o join o.bookings ob join ob.images oi WHERE oi.captureDateTime > :start")
     Page<Offender> getOffendersWithImagesCapturedAfter(@Param("start") LocalDateTime start, Pageable pageable);
+
+    @Query(value = "SELECT OFFENDER_HEALTH_PROBLEM_ID.nextval FROM dual", nativeQuery = true)
```

Comment:
Why? - can't you just add sequence to entity?

Reply:
have changed to using sequence